README.md

```diff
@@ -1,8 +1,126 @@
+
+- Split app model: firebase
+- Phone sensors: GPS
+- User support
+- Database access
+- Automated test suite: 80% code coverage
+
+# Build Status
 
 [![Build Status](https://travis-ci.org/basbeu/theSofties.svg?branch=master)](https://travis-ci.org/basbeu/theSofties)
 
 <a href="https://codeclimate.com/github/basbeu/theSofties/maintainability"><img src="https://api.codeclimate.com/v1/badges/ca33d5d2d2e82d260e05/maintainability" /></a>
 
 <a href="https://codeclimate.com/github/basbeu/theSofties/test_coverage"><img src="https://api.codeclimate.com/v1/badges/ca33d5d2d2e82d260e05/test_coverage" /></a>
 
+# Features
+
+######Completed
+
+- users can create accounts, login, edit their profile and reset their password
+- users can see a list of favors, a detailed description of a favor and can ask for a new one and set a deadline
+- there are several categories favors can be ordered upon, additionally they can be ordered based on location, most recent and searched by name or description
+
+###### Roadmap
+
+- users can add a profile picture and add media files to their favors
+- users can see the favors on a map around them with a small pin and a tag
+- users can chat about meeting up for completing an issue and receive notifications
+- users can "pay" using tokens and mark a favor as completed
+- users can request multiple people for a favor
+- users can see a history of the favors they have asked for and completed
+
+# Scrum
+
+### Sprint 1
+
+##### Scrum Master: Jeremy, presenter: Bastien
+
+- git setup, filling backlog, authentification and rudimentary database and UI
+
+### Sprint 2
+
+##### Scrum Master: Nicolas, presenter: Andrea
+
+- build a professional and concise database structure with firestore
+- expand UI & profile functionality
+- added basic favor capability that communicated with firestore using observables
+
+<img src="https://mail.sanchez.at/files/favor-screen-1/favor-screen-1.png" alt="drawing" width="200"/>
+
+### Sprint 3
+
+##### Scrum master: Charline, presenter: Jeniffer
+
+- favor list with UI for searching & sorting, as well as asking for a new favor
+- location, distance, category, etc. are recorded alongside favor
+
+
+
+# Tests
+
+right now there is only a very limited amount of tests
+
+This will be changed in Sprint 3
+
+``` bash
+code for running the single line command in the command line goes here
+```
+
+## API Reference
+
+##### Database
+
+###### Users
+
+[Please insert the most important API functionality with description and example here for database]
+
+```java
+code example goes here
+```
+
+###### Favors
+
+[Please insert the most important API functionality with description and example here for database]
+
+```
+code example goes here
+```
+
+## Installation
+
+For installing this project in the development environment you will need:
+
+- Gradle
+- Java v.1.8
+- Android SDK v.28
```

Comment:
technically our min SDK is 15. See gradle app file. The target is what we will run on if it is available. Our app sould work on SDK 15